src/playbacks/html5_video/html5_video.js

```diff
@@ -408,6 +408,8 @@ export default class HTML5Video extends Playback {
 
 HTML5Video._canPlay = function(type, mimeTypesByExtension, resourceUrl, mimeType) {
   var extension = (resourceUrl.split('?')[0].match(/.*\.(.*)$/) || [])[1]
+  extension = (extension || "").toLowerCase()
+  
   var mimeTypes = mimeType || mimeTypesByExtension[extension] || []
```

Comment:
Hi @sjlu

Well spotted thanks for the PR.

I think changing this line to
```javascript
var mimeTypes = mimeType || (extension && mimeTypesByExtension[extension.toLowerCase()]) || []
```
would be better, because we shouldn't be trying to access `mimeTypesByExtension[undefined]`